### PR TITLE
Add WebView versions for HTMLMediaElement API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for WebView Android for the `HTMLMediaElement` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.1.8).  The collector obtains results based upon the latest WebView version (to determine if it is supported), then version numbers are copied from Chrome Android.  Results are manually confirmed for accuracy.

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/HTMLMediaElement

### CODE REVIEW

File: api/HTMLMediaElement.json [hunk 1]
```diff
@@ -357,7 +357,7 @@
               "version_added": "4.0"
             },
             "webview_android": {
-              "version_added": "43"
+              "version_added": true
```
Comment (foolip) [inline api/HTMLMediaElement.json:360]: Why this change? A lot of the "43" in this file are wrong and would be fixed by testing the instance, maybe fix all of those first to ensure that no known-wrong data is being mirrored or overwritten in this PR? We should be able to fill in a real value here after the 43s and true values elsewhere are fixed.

Comment (vinyldarkscratch) [reply api/HTMLMediaElement.json]: I actually don't know why I did this.  I think I might've been thinking "I'll set it to `true` and fix these later", but we've got an issue open to track these down.  Reverting these changes so that this PR is only increasing the amount of real values, not decreasing.

Comment (foolip) [reply api/HTMLMediaElement.json]: Yep, let's get back to these in https://github.com/mdn/browser-compat-data/issues/7843.

File: api/HTMLMediaElement.json [hunk 2]
```diff
@@ -1003,7 +1003,7 @@
               "version_added": "5.0"
             },
             "webview_android": {
-              "version_added": "49"
+              "version_added": false
```
Comment (foolip) [inline api/HTMLMediaElement.json:1006]: Yep, confirmed by crbug.com/521319. I'll send a PR to link that issue after merging this.

File: api/HTMLMediaElement.json [hunk 3]
```diff
@@ -3648,7 +3651,7 @@
...
```
Comment (foolip) [inline api/HTMLMediaElement.json:3654]: Confirmed by crbug.com/589500.

**Review by foolip — APPROVED** [review]
